paket.dependencies

```diff
@@ -1,12 +1,23 @@
 redirects: on
 source https://www.nuget.org/api/v2/
 
+// We want to depend on FSharp.Core 4.0.0.1, but not require
+// that version in the generated nuget package (see paket.template).
+//
+// If we just specify "FSharp.Core 4.0.0.1" then the nuget package
+// has an dependency on that *exact* version.
+//
+// So, we specify 4.0.0.1 as a minimum bound, and set lowest_matching
+// so it won't be updated automatically to a newer one by 'paket update'.
+nuget FSharp.Core >= 4.0.0.1 lowest_matching: true
+
```

Comment:
FWIW, I just run a build with the NuGet flag to create a package locally. Indeed, it's now requiring FSharp.Core >= 4.0.0.1

![image](https://cloud.githubusercontent.com/assets/287532/24993721/413fee5c-2030-11e7-89bd-d030f471e8e1.png)

👍